transportable-udfs-test/transportable-udfs-test-generic/src/main/java/com/linkedin/transport/test/generic/GenericConverters.java

```diff
@@ -28,12 +28,12 @@
 import java.util.Map;
 
 
-public class GenericWrapper {
+public class GenericConverters {
```

Comment:
Are we replacing `Generic` with `Test`? That change needs to be global.

Reply:
Let's do that in seperate PR.

Reply:
Right now, I can see Generic is a common prefix for all classes in transport.test.generic package. If we only want to change GenericConverts/GenericTypeFactory/GenericUDF to TestConverts/TestTypeFactory/TestUDF, I don't think it is good enough. (https://github.com/linkedin/transport/tree/master/transportable-udfs-test/transportable-udfs-test-generic/src/main/java/com/linkedin/transport/test/generic). Strongly recommend to update that in a seperate PR after we made a final decision.